src/pages/Filing/FilingHome.tsx

```diff
@@ -23,8 +23,8 @@ function Home(): ReactElement {
     <div id='filing-home'>
       <Layout.Main id='main' layout='2-1' bleedbar classes='main-layout'>
         <Hero
-          heading='Get started filing your small business lending data'
-          subheading='Covered financial institutions are required to compile, maintain, and report information about small business lending.'
+          heading='Get started filing your lending data'
+          subheading='Covered financial institutions and voluntary reporters to maintain, report, and publicly disclose information about lending.'
```

Comment:
I see you updated based on the figma, but I don't think it's right. Don't we still need `are required` here in this subheading, the sentence sounds weird without it? Also, I don't think covered financial institutions are required to publicly disclose information: that's what we do. They just need to report it.
![Screenshot 2024-04-02 at 1 46 26 PM](https://github.com/cfpb/sbl-frontend/assets/19983248/70d2a694-f031-4c4c-9350-cac16a8b5e50)

Reply:
I see on our website we say something about publicly disclosing stuff, but I think the "rule" subject there is doing some heavy lifting. If include the "are required" then I'm fine keeping the rest the same. 👍

![Screenshot 2024-04-02 at 1 53 32 PM](https://github.com/cfpb/sbl-frontend/assets/19983248/fb4264a6-730f-4bc6-bd2b-da8aaae971a1)

Reply:
Updated hero content per the design file. 

`Covered financial institutions are required to maintain, report, and publicly disclose information about lending to the CFPB.`